docs/deployment/benchmark.md

```diff
@@ -0,0 +1,65 @@
+# Benchmarking your Invenio instance
+
+## Populate your instance
```

Comment:
I guess is an optional step to engage people. They have the full experience of a deployed instance with some data there so they can see them in the UI. Personally I perceive the deployment as the `containerized` version but without the warning "You should not put this in production" :)

Reply:
> how to add fake random records to the real deployed system in the benchmarking page?

I think I did not pass the message along. But I wouldn't do this test in a production/real instance. Mainly because it is highly likely to take it down, several times. For example, we managed to render the DB to not respond for some time (~20mins) until the DBAs acted on it... It is quite a hardcore test.

On this, my idea was:

- If you have a production instance, you might not want to make an effort to "duplicate" your data, nor use the prod DB/ES for stress test. However, you do want to create a sensible amount of records that represent the amount of records in prod. E.g. ES does not search as fast with 1 record than with 3M. It might need improvements on the shards rather than the application architecture, but still this test would point out slow parts.
- You do not have a production instance. So you need to create "dummy" records to simulate what you would expect to have.

On naming on the section, I am open to change it. But I would still mention this here :) WDYT?

Reply:
What about "Creating demo records" and explain a bit what you said about the purpose behind it? I think the 2 examples you just mentioned are quite useful for people :)

Reply:
Adding some of the context/explanation/motivation you just provided would be helpful and changing the filename + main title from "Benchmarking your InvenioRDM instance" to "Stress testing your InvenioRDM instance" (or "Load testing" or "verifying" or something like that :) ) would clarify the context.